ansible/roles/cli/tasks/deploy.yml

```diff
@@ -26,3 +27,7 @@
   when: cli_installation_mode == "local"
 
 - include: download_cli.yml
+  when: cli_installation_mode != "remote_redirect"
```

Comment:
My gut says this should be `cli_installation_mode == "remote"` because selecting local install is usually because the download is broken for a particular platform.  Of course, I'm not sure local is functionally complete in the configuration.  But, more to the point, we should separate making the CLI available for download via NGinx from sourcing the binary for /wsk/bin for local use.